src/components/atoms/textInput/InputText.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './InputText.css';
+
+export default function InputText(props) {
+  return (
+    <input className="input" type="text" placeholder={props.placeholder} ref={props.searchName}/>

```

Comment:
정말 nit 하긴 한데 다음과 같이 나눠 보는 것은 어떨까요?? 
가독성에 더 도움이 될 듯 해서요!

```suggestion
    <input
      className="input"
      type="text"
      placeholder={props.placeholder}
      ref={props.searchName}
    />

```